cmd/zt_scenario_helpers_for_test.go

```diff
@@ -285,7 +285,7 @@ func (scenarioHelper) generateBlobsFromList(c *chk.C, containerURL azblob.Contai
 	for _, blobName := range blobList {
 		blob := containerURL.NewBlockBlobURL(blobName)
 		cResp, err := blob.Upload(ctx, strings.NewReader(data), azblob.BlobHTTPHeaders{},
-			nil, azblob.BlobAccessConditions{})
+			nil, azblob.BlobAccessConditions{}, azblob.DefaultAccessTier)
```

Comment:
>azblob.DefaultAccessTier [](start = 39, length = 24)

To confirm, the default value is no value, right? In other words we don't send any value to the service?

I'm asking since the cx's account has a default tier, so we should let the service decide the default.

Reply:
Yes.  
```var DefaultAccessTier AccessTierType = ""```
That's how it is declared and it is equivalent to AccessTierNone